### PR TITLE
Issue #DP-392 fix : Reject comment pop up gets closed on click over t…

### PR DESCRIPTION
**Pull request by vaibhavbhuva**

### CODE REVIEW

File: src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html [hunk 1]
```diff
@@ -212,7 +212,7 @@ <h4 class="sb-pageSection-title six">{{resourceService.frmelmnts.lbl.textbooks}}
               <td *ngIf="currentNominationStatus === 'Initiated' || currentNominationStatus=== 'Pending'">{{data.totalSampleContent || 0 }}</td>
               <td>
                 <div *ngIf="selectedContentTypes.length">
-                <div *ngIf="(data.totalSampleContent === undefined || data.totalSampleContent === 0) && canNominate()" class="sb-checkbox sb-checkbox-primary d-inline-block">
+                <div *ngIf="canNominate()" class="sb-checkbox sb-checkbox-primary d-inline-block">
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html:215]: @vaibhavbhuva If there are any samples are uploaded against the textbook, can we have that checkbox disabled?

File: src/app/client/src/app/modules/program/components/create-program/create-program.component.html [hunk 2]
```diff
@@ -358,7 +359,13 @@ <h4 class="sb-pageSection-title">{{resource.frmelmnts.lbl.createNewProgram}}</h4
 													<input type="checkbox" id="checkitem_{{data.identifier}}"
 														(change)="onCollectionCheck(data, $event.target.checked)"
 														[checked]="collectionListForm.value.pcollections.indexOf(data.identifier) != -1"
-														name="checkitem">
+														name="checkitem"
+														appTelemetryInteract
+														[telemetryInteractEdata]="getTelemetryInteractEdata('select-textbook', 'click', 'create-program',{type:'reference',identifier:data.identifier})"
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/program/components/create-program/create-program.component.html:364]: what does the select-textbook represents? If this is the action then it should be something related to program-document-upload or something.

**Comment by twsvaishali** [comment]
@vaibhavbhuva Please try to send different PR for different issues. Else just for 1 change all of your solved issues will be in pending state

File: src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html (hunk 1 above)
Comment (vaibhavbhuva) [reply src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html:215]: @twsvaishali Yes, I will make it as a disable state if samples are uploaded.

File: src/app/client/src/app/modules/program/components/create-program/create-program.component.html (hunk 2 above)
Comment (vaibhavbhuva) [reply src/app/client/src/app/modules/program/components/create-program/create-program.component.html]: @twsvaishali 

@venkanagouda-k already has done this telemetry event. So, I have reverted my code.

**Pull request closed without merging**